README.md

```diff
@@ -1,9 +1,32 @@
-Meetup #1
+High To Low workshop #1
```

Comment:
Nesúhlasím. Už sa viac ľudí poplietlo na názve. Myslím, že by sme to mali volať Rust meetup. (Implicitne v `rust-bratislava`.)